src/EditorFeatures/CSharpTest/Completion/CompletionProviders/ExtensionMethodImportCompletionProviderTests.cs

```diff
@@ -701,53 +702,6 @@ public void M(int x)
                  inlineDescription: "Foo");
         }
 
-        [InlineData(ReferenceType.Project)]
-        [InlineData(ReferenceType.Metadata)]
-        [Theory, Trait(Traits.Feature, Traits.Features.Completion)]
-        public async Task TestConflictingInternalExtensionMethods_NoIVT_InReference(ReferenceType refType)
```

Comment:
don't remove tests.  just update to expected behavir.

Reply:
Didn't remove this. Just moved it to the bottom and made some changes

Reply:
ah ok.  in the future, don't move tests if possible :)